config/locales/en.yml

```diff
@@ -27,5 +27,7 @@ en:
   lists:
     list:
       empty: This list is empty
```

Comment:
I think we can change this message to "We don't have any curated list yet." and keep the message of show action.